src/node/internal/process.ts

```diff
@@ -62,7 +64,12 @@ export const env = new Proxy({}, {
   }
 });
 
+export function getBuiltinModule(id : string) : any {
```

Comment:
nit: fix spaces

Reply:
For stuff like this... using the "Add a suggestion" option in the github editor is easier.